v1/ast/oracle/oracle.go

```diff
@@ -225,11 +327,9 @@ func walkToFirstOccurrence(node ast.Node, needle ast.Var) (match *ast.Term) {
 }
 
 func findContainingNodeStack(module *ast.Module, pos int) []ast.Node {
-
```

Comment:
My formatter removes these can I think it's best to be consistent so have left the removals in.